validate/validate.proto

```diff
@@ -514,45 +562,49 @@ message StringRules {
     // values
     repeated string not_in = 11;
 
+    // Omitempty specifies that the validation rules of this field should be
+    // evaluated only if the field is not empty
+    optional bool omitempty = 12;
```

Comment:
I know that protoc-gen-validate is in beta, but it really would be nice to make this backwards-compatible (even though in practice it matters less since these options aren't sent over the wire). Ie instead of adjusting all the tags below, make this tag 23

Reply:
Agreed. Please do not renumber these fields.

Reply:
Ok